doc/source/release-notes/Percona-Server-5.7.27-30.rst

```diff
@@ -0,0 +1,119 @@
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date|. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.27-30/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.27
+<https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-27.html>`_ and includes
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+	
+All software developed by Percona is open-source and free.
+
+.. note:: 
+
+   If you're currently using |Percona Server| 5.7, Percona recommends upgrading
+   to this version of 5.7 prior to upgrading to |Percona Server| 8.0.
+
+
+Bugs Fixed
+================================================================================
+
+- Parallel doublewrite buffer writes must crash the server on an I/O error 
+  occurs. Bug fixed :psbug:`5678`.
+
+- On a server with two million or more tables using foreign keys and AUTOINC columns, the shutdown may take a measurable 
+  length of time. Bug fixed :psbug:`5639`. (Upstream :mysql:`95895`)
+
+- If large pages are enabled on the MySQL side, the maximum size for 
+  :variable:`innodb_buffer_pool_chunk_size` is effectively limited to 4GB. Bug 
+fixed :psbug:`5517`. (Upstream :mysql:`94747`)
+
+- The TokuDB hot backup library continually dumps TRACE information to the 
+  Server error log. The user cannot enable or disable the dump of this 
+information. Bug fixed :psbug:`4850`.
+
+- Key rotation is redesigned to allow ``SELECT 
+  rotate_system_key("percona_redo")``. The currently used key version is 
+displayed in the :variable:`innodb_redo_key_version` status. Bug fixed 
+:psbug:`5565`.
+
+- The TokuDBBackupPlugin is optional at cmake time. Bug fixed :psbug:`5748`.
+
+- A multi-table ``DELETE`` with a foreign key breaks replication. Bug fixed :psbug:`3845`.
+
+- A ``TRUNCATE`` with any table and interfacing with Adaptive Hash Index (AHI) can cause server stalls due to the interaction with AHI, whether the AHI is enabled or not. Bug fixed 
+:psbug:`5576`. (Upstream :mysql:`94610`)
+
+- In specific configurations and with :variable:`log_slow_verbosity` set to log 
+  InnoDB statistics, memory usage increases while running a stored procedure. 
+Bug fixed :psbug:`5581`.
+
+- `Threadpool <https://www.percona.com/doc/percona-server/5.7/performance/threadpool.html>`__ functionality to track network I/O was disabled.  Bug fixed :psbug:`5723`.
+
+- When Adaptive Hash Index (AHI) is enabled or disabled, there is an AHI overhead during 
+  DDL operations. Bug fixed :psbug:`5747`.
+
+- An instance started with the default values but setting the `redo log to 
+  encrypt 
+<https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#redo-log-encryption>`__ 
+without specifying the keyring plugin parameters does not fail or throw an 
```

Comment:
:ref:`data_at_rest_encryption` instead of html link

Reply:
done.